packages/design-system/src/interfaces/buttons.d.ts

```diff
@@ -20,6 +21,8 @@ export interface IButtonProps
   renderIcon?: string | React.ReactNode;
   ref?: React.Ref<HTMLButtonElement> | React.Ref<React.Component<IButtonProps>>;
   secondary?: boolean;
+  text?: boolean;
+  iconPosition?: ButtonIconPosition;
```

Comment:
1. no `kind` property here 
2. you can add note about deprecated properties here for hints in code editor (example in tooltip.d.ts)
3. add `export type ButtonKind = "primary" | ... | "text"` above

Reply:
2ebe102873729d6f5b0b13d7b05f8cec0d521e16